tox.ini

```diff
@@ -63,14 +63,13 @@ commands =
     echo "Troubleshooting guide: https://docs.plasmapy.org/en/latest/contributing/doc_guide.html#troubleshooting"
 deps = -r{toxinidir}/requirements.txt
 
-[testenv:build_docs_linkcheck]
+[testenv:linkcheck]
 changedir = {toxinidir}
 setenv =
     HOME = {envtmpdir}
     PYDEVD_DISABLE_FILE_VALIDATION=1
 commands =
-    sphinx-build docs docs{/}_build{/}html -W -n --keep-going -b linkcheck {posargs}
-    cat docs/_build/html/output.txt
+    sphinx-build docs docs{/}_build{/}html -W -n -q --keep-going -b linkcheck {posargs}
```

Comment:
The `-q` flag makes it so that only broken links and redirects show up in the linkcheck output.  Without it, the results for every link that's checked gets outputted to the screen.